emission/core/wrapper/entry.py

```diff
@@ -73,6 +73,8 @@ def _getData2Wrapper():
             "manual/mode_confirm": "userlabel",
             # user confirmation of the travel purpose
             "manual/purpose_confirm": "userlabel",
+            #user response to the survey questions
+            "manual/survey_response": "userlabel",
```

Comment:
I think that @asiripanich uses `manual/survey_confirm` instead. Maybe we could come up with a standardized nomenclature. Also, should we have separate labels for the initial demographic survey and the trip-level surveys?

Reply:
Okay, so the current nomenclature follows the "statement_confirm" pattern? I can switch to that. 
What do you mean by separate labels for this?

Reply:
@jruzekowicz don't change to it until we hear from @asiripanich

>  What do you mean by separate labels for this?

Right now, we are only displaying a survey when the user starts the onboarding process. We might want to have trip-level surveys, similar to https://nextcloud.damajash.org/s/CRZg4zQY2gKWMnF as well, in which case, we may want to have a different key for that.

You don't need to make that different key right now, but we may want to call this `manual/demo_survey_confirm` or something to distinguish from trip-level surveys.

Reply:
Hi Shankari, personally I don't mind `survery_response`. We do the initial survey differently than you do. We use an in-app browser to open an online survey form instead of using Enketo. However, for the trip-level surveys, we use Enketo because it allows us to capture the responses and use them to display on the UI and the responses to be edited without internet.

Reply:
@shankari Would it be better suited to call this something such as "Itinerum_survey_response"?